content/en/docs/tutorials/configuration/configure-redis-using-configmap.md

```diff
@@ -9,37 +9,37 @@ weight: 30
 
 <!-- overview -->
 
-This page provides a real world example of how to configure Redis using a ConfigMap and builds upon the [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) task. 
+After you [configure a Pod to use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/), you can then use a ConfigMap to configure Redis.
 
+## What you'll learn
```

Comment:
Is there a reason for moving away from the conventional Objectives section heading?